pkgs/tools/networking/par2cmdline-turbo/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "par2cmdline-turbo";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "animetosho";
+    repo = "par2cmdline-turbo";
+    rev = "v${version}";
+    sha256 = "sha256-+p6EeWn+tgZFRWSK4feQ4TIXJP+2iIcu9Ng0T6JNRCE=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  meta = with lib; {
+    homepage = "https://github.com/animetosho/par2cmdline-turbo";
+    description = "par2cmdline × ParPar: speed focused par2cmdline fork";
+    license = licenses.gpl2Plus;
+    maintainers = [ ];
```

Comment:
You need to add yourself as a maintainer https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

Reply:
Added